pkg/monitor/cluster/clusterflagsandbanner.go

```diff
@@ -0,0 +1,46 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+)
+
+const (
+	operatorFlagMetricsTopic  = "cluster.nonstandard.operator.featureflags"
+	supportBannerMetricsTopic = "cluster.nonstandard.contact.support.banner"
+)
+
+func (mon *Monitor) emitOperatorFlagsAndSupportBanner(ctx context.Context) error {
+	cs, err := mon.listAROClusters(ctx)
+	if err != nil {
+		return err
+	}
+
+	for _, cluster := range cs.Items {
+		if cluster.Spec.OperatorFlags != nil {
+			defualtFlags := api.DefaultOperatorFlags()
+			nonStandardOperatorFlags := make(map[string]string)
+			//check if the current set flags matches the default ones
+			for flag, status := range cluster.Spec.OperatorFlags {
```

Comment:
do you think we should loop over the defaultFlags rather than the current cluster flags? If the cluster flags are missing some then it won't be included in the loop or emitted via the monitor

Reply:
I agree this should be flipped and add a check for when the cluster flags are missing a default flag key.